Initial scraper

ondenman opened the pull request:
Initial scraper for Samoa 2016 term.

## [Scraper Change checklist](https://github.com/everypolitician/everypolitician/wiki/Scraper-Change-checklist)
* [x] 1. scraper is on Morph.io under the "everypolitician-scrapers" group?
* [x] 2. scraper's GitHub "Website" link points at morph.io page?
* [x] 3. scraper is set to auto-run?
* [x] 4. scraper is archiving?
* [x] 5. legislature has a scraper webhook set?

## [Gemfile checklist](https://github.com/everypolitician/everypolitician/wiki/Gemfile-checklist/)
* [x] 1. all links are secure
* [x] 2. links to Github use `github:` protocol, not simply `git:`
* [x] 3. formatting is consistent with our normal Rubocop setup

ondenman committed:
Add Rubocop config

ondenman committed:
Add Gemfile

ondenman committed:
Add Rubocop Rake task

ondenman committed:
Configure Travis to run Rubocop

ondenman committed:
Scrape to archive

The source is a single page. This commit adds the page to archive.

tmtmtmtm commented on scraper.rb:
```diff
@@ -0,0 +1,23 @@
+require 'scraped'
+require 'require_all'
+
+require_rel 'lib'
```
There doesn't seem to _be_ a `lib` directory to require yet…

tmtmtmtm reviewed (changes requested):
the description of this PR should make it clear that this is just the initial framework for archiving the list page, rather than actually doing any parsing etc yet.

tmtmtmtm committed:
Make 'rubocop' the default rake task

tmtmtmtm committed:
Use default rake task on travis

Rather than explicitly specifying rubocop, just fall back on a simple
'rake default', which now includes rubocop

tmtmtmtm committed:
Speed up travis

We probably don't explicitly _need_ the 'sudo: false' as this is a new
repo, but as some of our repos _are_ old enough, it's simpler to just be
consistent everywhere.

tmtmtmtm reviewed (approved):
@ondenman I've made a few tweaks to how travis runs, and making `rubocop` a default rake task, but otherwise :+1:
